locales-pending/premium.ftl

```diff
@@ -41,6 +41,11 @@ exposure-chart-legend-value-nr = { $nr }x
 
 # Here's What We Fixed Progress Card
 
+exposure-card-company-logo = Company logo
+exposure-card-company = Company domain
+exposure-card-exposure-type = Exposure types
```

Comment:
Slight discrepancy here between "type" and "types".
Not sure if we should pluralize the ID or singular-ize the string.

Reply:
Updated in [6ea0351](https://github.com/mozilla/blurts-server/pull/3246/commits/6ea0351f6dca99ec79fed1791ee4877c958ab0c5).